_pages/attributes.md

```diff
@@ -173,7 +173,7 @@ When the user's identity was last verified (or empty if it has never been verifi
 
 Seconds since the Unix Epoc
 
-Requires the `profile` or `profile:verified_at` scope.
+Requires the `profile` or `profile:verified_at` scope. To get the attribute with IAL1, the `profile:verified_at` scope must be requested.
```

Comment:
Maybe we should just simplify to

```suggestion
Requires the `profile:verified_at` scope.
```

and we updated https://developers.login.gov/attributes/ as well?

Reply:
I think it's ok to simplify, but did you mean to link to a different URL? This change is updating the page you linked 🙂

Reply:
whoooops I meant the OIDC page, the description of `verified_at` towards the bottom

https://developers.login.gov/oidc/

Reply:
Changed OIDC page to use the same language.

Reply:
sweeeeeeet thanks